portfolio/src/main/webapp/script.js

```diff
@@ -57,9 +72,18 @@ function deleteComments() {
   fetch(request).then(getComments());
 }
 
+function checkPassword() {
+  var adminInput = document.getElementById("admin-password").value;
+  if (adminInput == "azby88") {
+    document.getElementById("removeAllComments").style.visibility = "visible";
+  } else {
+    document.getElementById("removeAllComments").style.visibility = "hidden";
+  }
+}
+
 // CHARTS
 google.charts.load('current', {'packages':['geochart'],
-'mapsApiKey': 'AIzaSyCP-FXbugyYntkSsiE4hQqIAEKh4Li8_ow'
+'mapsApiKey': 'AIzaSyCP-FXbugyYntkSsiE4hQqIAEKh4Li8_ow' 
```

Comment:
same here, please consider move this key to the API key file :)

Reply:
Thanks Mostafa, this will be added to the API key file!